File: module-bet/src/main/java/com/xtree/bet/bean/request/im/GetBetListReq.java
package com.xtree.bet.bean.request.im;

import java.util.List;

public class GetBetListReq {

    /**
     * 确认投注状态清单.
     * 1 = Pending 待处理
     * 2 = Confirmed 已确认
     * 3 = Rejected (refers to Danger Cancel) 已拒绝 (危
     * 险球取消)
     * 4 = Cancelled 已取消
     */
    public List<Integer> BetConfirmationStatus;
    public String Token;
    public String MemberCode;
    public String startDate;
    public String EndDate;
    public String TimeStamp;
    public int SourceWallet;
    /**
     * 指出每页记录数.
     * 30, 50, 100
     * 默认 = 30
     */
    public int PageRecords;

    /**
     * 指出返回的页数
     */
    public int Page;


}

File: module-bet/src/main/java/com/xtree/bet/bean/request/im/GetBetListReq.kt
package com.xtree.bet.bean.request.im

class GetBetListReq {
    var api = "getBetList"
    var method = "post"
    var format = "json"
    var LanguageCode = "CHS"//使用会员语言

    /**
     * 确认投注状态清单.
     * 1 = Pending 待处理
     * 2 = Confirmed 已确认
     * 3 = Rejected (refers to Danger Cancel) 已拒绝 (危
     * 险球取消)
     * 4 = Cancelled 已取消
     */
    lateinit var BetConfirmationStatus: List<Int>

    //var StartDate: String? = null,
    //var EndDate: String? = null,
    //var SourceWallet: Int? = 0,

    var MemberCode: String = ""

    //后端统一处理
    var Token: String = ""
    //lateinit var TimeStamp: String
}
